scripts/custom_lint_checks.py

```diff
@@ -520,13 +567,27 @@ def _compare_different_args(
                                 not_needed_type_in_docstring)
 
     def check_single_constructor_params(self, class_doc, init_doc, class_node):
+        """Check single constructor Parameters
+
+        Args:
+            class_doc: The docstring class instance that represents a class's docstring.
+            init_doc:  The docstring class instance that represents a method's docstring, the method
+                here is the constructor method for the above class.
+            class_node: Node. Node for class definition in AST.
+        """
         if class_doc.has_params() and init_doc.has_params():
             self.add_message(
                 'multiple-constructor-doc',
                 args=(class_node.name,),
                 node=class_node)
 
     def _handle_no_raise_doc(self, excs, node):
+        """Check if there is no raise, then add a message.
```

Comment:
"Checks whether the raised exception in a function has been documented, add a message otherwise."

Reply:
done.